### PR TITLE
Remove RegistryConfiguration namespace field

### PR DESCRIPTION
**Pull request by jdartigalongue**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/765#" title="Author self-approved">jdartigalongue</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [jdartigalongue]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/alert/ctl_alert.go [hunk 1]
```diff
@@ -129,7 +129,6 @@ func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Comma
 	cmd.Flags().StringVar(&ctl.CfsslMemory, "cfssl-memory", ctl.CfsslMemory, "Memory allocation of CFSSL")
 	cmd.Flags().StringVar(&ctl.DesiredState, "alert-desired-state", ctl.DesiredState, "State of Alert")
 	cmd.Flags().StringVar(&ctl.Registry, "registry", ctl.Registry, "Name of the registry to use for images")
```
Comment (rrati) [inline pkg/alert/ctl_alert.go]: Should we expand this description to state name and namespace of the registry?

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/765#" title="Author self-approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/765#pullrequestreview-274859725" title="Approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [jdartigalongue,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/765#" title="Author self-approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/765#pullrequestreview-274943221" title="LGTM">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [jdartigalongue,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
New changes are detected. LGTM label has been removed.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/765#" title="Author self-approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/765#issuecomment-521398488" title="LGTM">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [jdartigalongue,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**